monkey/tests/unit_tests/monkey_island/cc/environment/test_utils.py

```diff
@@ -20,38 +20,26 @@ def test_path(tmpdir):
     return path
 
 
-def test_create_secure_directory__parent_dirs(test_path_nested):
-    create_secure_directory(test_path_nested, create_parent_dirs=True)
-    assert os.path.isdir(test_path_nested)
-
-
 def test_create_secure_directory__already_created(test_path):
     os.mkdir(test_path)
     assert os.path.isdir(test_path)
-    create_secure_directory(test_path, create_parent_dirs=False)
+    create_secure_directory(test_path)
 
 
 def test_create_secure_directory__no_parent_dir(test_path_nested):
     with pytest.raises(Exception):
-        create_secure_directory(test_path_nested, create_parent_dirs=False)
-
-
-@pytest.mark.skipif(is_windows_os(), reason="Tests Posix (not Windows) permissions.")
-def test_create_secure_directory__perm_linux(test_path_nested):
-    create_secure_directory(test_path_nested, create_parent_dirs=True)
-    st = os.stat(test_path_nested)
-    return bool(st.st_mode & stat.S_IRWXU)
+        create_secure_directory(test_path_nested)
 
 
 @pytest.mark.skipif(not is_windows_os(), reason="Tests Windows (not Posix) permissions.")
 def test_create_secure_directory__perm_windows(test_path):
-    import win32api  # noqa: E402
-    import win32security  # noqa: E402
+    import win32api
+    import win32security
```

Comment:
Same question about `# noqa`

Reply:
We don't, isort doesn't touch them most likely because they're inside an `if` block.